x/erc20/keeper/proposals.go

```diff
@@ -58,6 +58,7 @@ func (k Keeper) RegisterCoin(ctx sdk.Context, coinMetadata banktypes.Metadata) (
 
 // verify if the metadata matches the existing one, if not it sets it to the store
 func (k Keeper) verifyMetadata(ctx sdk.Context, coinMetadata banktypes.Metadata) error {
+	// Metadata cant be invalid since its checked that bank supply is available
```

Comment:
I don't understand this comment, can you rephrase it?

Reply:
The comment was wrong.
Deleted the comment and added `coinMetadata.Validate()` check

Reply:
Isn't the metadata validated already in the proposal types?

Reply:
Oh, thats correct. Ill delete this check since its unnecessary